modules/cnf-configuring-workload-hints.adoc

```diff
@@ -0,0 +1,84 @@
+
+:_content-type: CONCEPT
+[id="cnf-configuring-workload-hints_{context}"]
+= Configuring workload hints
+
+Workload hints simplify the fine-tuning of performance to industry sector settings. Instead of a “one size fits all” approach, workload hints can cater to usage patterns such as placing priority on:
+
+* Low latency
+* Real-time capability
+* Efficient use of power
+
+In an ideal world, all of those would be prioritized: in real life, some come at the expense of others. The Node Tuning Operator is now aware of the workload expectations and better able to meet the demands of the workload. The cluster admin can now specify into which use case that workload falls. The Node Tuning Operator uses hints to fine tune the performance settings for the workload.
+
+The environment in which an application is operating influences its behavior. For a typical data center with no strict latency requirements, only minimal default tuning is needed that enables CPU partitioning for some high performance workload pods.
+For data centers and workloads where latency is a higher priority, measures are still taken to optimize power consumption.
+The most complicated cases are clusters close to latency-sensitive equipment such as manufacturing machinery and software-defined radios. This last class of deployment is often referred to as Far edge. For Far edge deployments, ultra-low latency is the ultimate priority, and is achieved at the expense of power management.
+
+[NOTE]
+====
+The following workload hints can be configured manually or created using the Performance Profile Creator. See "Creating a performance profile" for further information.
+====
+
+[cols="1,1,1,1",options="header"]
+|===
+    | `PerformanceProfile` setting| Hint | Environment | Description
+
+    | Default
+    a|[source,terminal]
+----
+workloadHints:
+highPowerConsumption: false
+realtime: false
+----
+    | High throughput cluster without latency requirements
+    | Performance achieved through CPU partitioning only.
+
+
+
+    | Low-latency
+    a|[source,terminal]
+----
+workloadHints:
+highPowerConsumption: false
+realtime: true
+----
+    | Regional datacenters
+    | Both energy savings and low-latency are desirable: compromise between power management, latency and throughput.
+
+
+    | Ultra-low-latency
+    a|[source,terminal]
+----
+workloadHints:
+highPowerConsumption: true
+realtime: true
+----
+    | Far edge clusters, latency critical workloads
+    | Optimized for absolute minimal latency and maximum determinism at the cost of increased power consumption.
+
+|===
+
+.Procedure
+
+. Create a `PerformanceProfile` appropriate for the environment's hardware and topology as described in previous sections. Then adjust the profile to match the expected workload. In this example, we tune for the lowest possible latency.
+
+. Add the `highPowerConsumption` and `realtime` workload hints and set them both to true as indicated by the above table.
```

Comment:
in the .md is realTime:
`| realTime | RealTime defines if the node should be configured for the real time workload. | *bool | false |`

but in the yaml is realtime. Not sure why highPowerConsumption is camelCase and realtime is not.

Reply:
If realtime in the yaml I will leave as is.